src/sdk/k4a.c

```diff
@@ -676,11 +682,19 @@ static k4a_result_t validate_configuration(k4a_context_t *device, const k4a_devi
         }
         else
         {
-            if (config->depth_delay_off_color_usec != 0 || config->synchronized_images_only)
+            if (config->depth_delay_off_color_usec != 0)
+            {
+                result = K4A_RESULT_FAILED;
+                LOG_ERROR("Both depth camera and color camera must be enabled to modify configuration of "
+                          "depth_delay_off_color_usec.",
+                          0);
+            }
+
+            if (config->synchronized_images_only)
             {
                 result = K4A_RESULT_FAILED;
-                LOG_ERROR("Both depth camera and color camera must be enabled to modify configuration of depth delay "
-                          "or synchronized image settings.",
+                LOG_ERROR("Both depth camera and color camera must be enabled to modify configuration of "
```

Comment:
Both depth camera and color camera must be enabled to ~~modify configuration of~~ **enable**